### DISCUSSION TITLE
Testing Plugins - How do I simulate multiple clients?

### DISCUSSION THREAD

**Discussion by jerbrisebois** [Impostor/Impostor#300] [Q&A] [answered]
I've been playing with plugin development but it's hard to test these with just a single client. I'd rather not have to bog down 4 other people's time to test and re-test. Is there any way to simulate having more players so I can test this? Or a reliable way to run multiple instances of the game? At the moment it seems Steam might be preventing me from launching the game twice.

**Comment by M4unz1** [comment, marked as answer]
You can try it with bluestacks (or any other kind of Android emulator)

**Reply by jerbrisebois** [reply]
> 
> 
> You can try it with bluestacks (or any other kind of Android emulator)

Worked perfectly! Thanks!

**Comment by Sanae6** [comment]
For people who are still looking to find a better solution for running more than one client, you can use the [Steam build](https://store.steampowered.com/sub/307391) with a Steam emulator (it's a crack/hack for steam games, so I'm not listing it here) or even better, use the [itch.io build](https://innersloth.itch.io/among-us) to run one or more clients. The [itch.io build](https://innersloth.itch.io/among-us) has no DRM so you're able to run it more than once without issue, making it the simpler way to run unmodded clients unlike the [Steam build](https://store.steampowered.com/sub/307391).
